.github/pull_request_template.md

```diff
@@ -0,0 +1,61 @@
+<!--- Provide a general summary of your changes in the Title above -->
+
+### Description
+
+<!--- Describe your changes in detail -->
+
+### Related Issue
+
+<!---
+This project only accepts pull requests related to open issues
```

Comment:
What's the rationale behind this requirement? In some other projects, I've accepted PRs without corresponding issues provided that they had good descriptions.

Reply:
This is common practice. So I adopted it. The reasoning behind such policies is the following: An issue allows you to discuss a change before implementing it. In contrast a PR already contains changes which the contributor might have put lots of effort into. Rejecting the PR because it contains unwanted changes (e.g. out of the project's scope) can be hard for the maintainer because of all the work that went into the PR. Furthermore the contributor will feel very demotivated, possibly even angry that he did all the work for nothing.

Reply:
On the other hand, I often find myself forking a project, poking around and trying out if I can easily fix a problem I've found, and then just submitting a PR if I succeeded. In such a case, I'd rather not be required to open an issue, describe the problem, open a PR, and describe the fix.

Anyway, it's not a big deal, I'm ok with this if it's common practice.

Reply:
Changed this, please have another look.

Reply:
I like it.